### DISCUSSION TITLE
Create VS Code extension

### DISCUSSION THREAD

**Discussion by AlexRMU** [josdejong/svelte-jsoneditor#281] [Ideas]
Create an editor for json/jsonc/json5/plain text
https://code.visualstudio.com/api/extension-guides/custom-editors

And for other editors?

**Comment by josdejong** [comment]
Thanks for sharing your idea. A VS Code plugin would be nice. Anyone interested in picking this up?

**Comment by AlexRMU** [comment]
https://marketplace.visualstudio.com/items?itemName=nickdemayo.vscode-json-editor (abandoned) and its forks:
- https://marketplace.visualstudio.com/items?itemName=jy-xlj.vscode-json-editor
- https://marketplace.visualstudio.com/items?itemName=pick68.vscode-json-editor-wsl
- https://marketplace.visualstudio.com/items?itemName=GregChamblin.vscode-json-editor
